src/freenet/clients/http/SecurityLevelsToadlet.java

```diff
@@ -698,10 +726,33 @@ public static void generatePasswordFormPage(boolean wasWrong, HTMLNode formNode,
 			content.addChild("#", l10nSec("enterPassword"));
 		}
 
-		formNode.addChild("input",
-		        new String[] { "type", "name", "size" },
-		        new String[] { "password", "masterPassword", "100" });
-
+		//Creates a table for password prompt and the confirmation box.
+		if(forUpgrade){
```

Comment:
The [style guide](https://google.github.io/styleguide/javaguide.html) (the link to which from the [contribution guidelines](https://github.com/freenet/fred/blob/next/CONTRIBUTING.md) was broken) suggests a space between the `if` and the open parenthesis.